.github/workflows/run-examples.yml

```diff
@@ -0,0 +1,46 @@
+name: examples
+
+on:
+  push:
+    # branches: [ master ]
```

Comment:
Are these commented out on purpose?

Reply:
It's a placeholder to specify particular branches to run this CI workflow for. As it's commented out now, the CI jobs will run for all branches.

Reply:
Now I remember that I used the existing `run-tests.yml`, which lists only the `master` branch, which prevents seeing the statuses of CI jobs for non `master` branches without creating a PR. I hope it makes sense.